package.json

```diff
@@ -2,43 +2,71 @@
   "name": "esutils",
   "description": "utility box for ECMAScript language tools",
   "homepage": "https://github.com/estools/esutils",
-  "main": "lib/utils.js",
+  "bugs": "https://github.com/estools/esutils/issues",
+  "main": "dist/esutils.min.js",
+  "module": "dist/esutils.esm.min.js",
   "version": "2.0.4-dev",
   "engines": {
     "node": ">=0.10.0"
   },
   "directories": {
-    "lib": "./lib"
+    "lib": "./dist"
   },
   "files": [
     "LICENSE.BSD",
     "README.md",
-    "lib"
+    "dist"
```

Comment:
this is a breaking change, since the dir name changed.

Reply:
Sure. I thought I would put it out there as such (in case other breaking changes were to be agreed upon anyways, such as use of `exports` as you mention). I can switch back to `lib` if the project maintainers wish, especially if there are no other breaking changes like `exports` being introduced, as would especially justify a switch to the clearer use of `dist` and which could, going forward, make it easier to avoid breaking changes (except to those using explicit `node_modules` paths, though even the [docs themselves](https://nodejs.org/api/esm.html#esm_main_entry_point_export) don't seem to consider that a semver violation).